acctz/acctz.proto

```diff
@@ -179,6 +175,9 @@ message GrpcService {
 
   // True, if truncation of payloads occurs due to an implementation
   // limitation in the originating service, any middleware, or the receiver.
+  // Truncation must only occur within a given API's message field (non
```

Comment:
Adjusted wording here to clarify that any truncation should only occur within the contents of a protobuf message field and not on the message/field boundaries themselves to allow for proper deserialization.

But I would like to raise the point that this is going beyond where we have been in classical accounting and is more analogous to full debug/tracing.  Since this is crossing such boundaries, I would likely propose additional configuration schema (or runtime gRPC APIs - however this is also going to be different among implementations) to toggle if this behavior should be enabled/disabled and not necessarily _always on_

Reply:
It seems like command accounting to me.  cli command accounting includes the command and args....  This is just more volumous.

Reply:
Yes - similar and likely varied today across various NBIs - CLI command accounting I would venture to bet is the most consistent whereas other APIs may not glean the full payloads or the complete wrapper framing w/o additional debugging/tracing turned on

gRPC message sizes as well as internal structures may end up also gating what comes in via 1 NBI may not easily be reflected out another and with truncation being the middle ground, would a partial message suffice in all cases?  Guessing probably enough to hope to correlate the transaction with the actual client side that invoked if tracing/accounting is performed there as well

Reply:
There is one impl. of grpc accounting (non-gnsi) and one of its services does truncate under certain circumstances, and not in the way your change describes.  I do not know how difficult it would be to implement this change, but it does seem useful to me.

Also, whether args are accounted could be controlled by a configuration knob.  or, control the size of args; or both.

Reply:
> Adjusted wording here to clarify that any truncation should only occur within the contents of a protobuf message field and not on the message/field boundaries themselves to allow for proper deserialization.

btw, I believe this really was the original intent, "do not make a pile of garbage when you decide not to send the 10gb config over this RPC, please trim it sanely, kthxbi!"